reference/changelog.md

```diff
@@ -24,6 +24,11 @@ redirect_from: /install/changelog.html
     </thead>
     <tbody>
 
+        <tr>
+            <th scope="row"><a href="https://github.com/pulumi/pulumi/blob/v0.16.6/CHANGELOG.md#0166-released-novemeber-28th-2018">0.16.6</a></th>
```

Comment:
I [fixed](https://github.com/pulumi/pulumi/pull/2249) the `novemeber` typo in CHANGELOG.md and didn't realize it was being referenced here. We should fix here as well.